packages/web3-eth-accounts/src/tx/baseTransaction.ts

```diff
@@ -565,4 +560,22 @@ export abstract class BaseTransaction<TransactionObject> {
 
 		return { r, s, v };
 	}
+
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	public static fromSerializedTx(
```

Comment:
If you want that new Tx class extending this BaseTx implements its own `fromSerializedTx`, make it abstract instead of static.

Reply:
but it should be static because other TX classes extend it and they have static methods `fromSerializedTx` and `fromTxData`